sdk/servicebus/service-bus/samples/javascript/advanced/deferral.js

```diff
@@ -49,8 +67,10 @@ async function sendMessages() {
         try {
           await sender.send(message);
           console.log("Sent message step:", data[index].step);
+          return;
```

Comment:
Interesting how these little discrepancies can creep in. Glad to see it matches the TS, even if it might not have mattered.

Reply:
Yeah I actually removed these "return" statements from both the TS and the JS in the lastest commits.